frontend/common/src/components/form/ErrorSummary.tsx

```diff
@@ -78,7 +76,7 @@ const ErrorSummary = React.forwardRef<
         {invalidFields.map((field) => (
           <li key={field.name}>
             <ScrollToLink to={field.name}>{field.label}</ScrollToLink>
-            {`${locale === "fr" && ` `}${field.message}`}
+            {`${` `}${field.message}`}
```

Comment:
This actually needs a colon here. The reason for the locale check is because there should be a space between the preceding word and the colon but not in FR.

Reply:
oh, I get it, that explains why there was a locale check there
just french things

https://github.com/GCTC-NTGC/gc-digital-talent/pull/5339/commits/335419a57f5a19e926875867501d5273b4028644